tests/classes/test_simplicialcomplex.py

```diff
@@ -27,6 +27,12 @@ def test_constructor(edgelist5, dict5, incidence5, dataframe5):
     with pytest.raises(XGIError):
         xgi.SimplicialComplex(1)
 
+    H = xgi.Hypergraph(edgelist5)
+    S_h = xgi.SimplicialComplex(H)
+
+    assert set(S_h.nodes) == set(H.nodes) == set(S_list.nodes)
+    assert set(S_h.edges) == set(S_list.edges)
```

Comment:
This is only checking edge IDs, so it's almost like checking the number of edges. 
Can you check their members by comparing `H.edges.members()` or `H._edge`?